pkg/stub/configmap.go

```diff
@@ -0,0 +1,188 @@
+package stub
+
+import (
+	"fmt"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	corev1 "k8s.io/api/core/v1"
+
+	v1 "github.com/openshift/api/samples/v1"
+	"github.com/openshift/cluster-samples-operator/pkg/util"
+
+	"github.com/sirupsen/logrus"
+)
+
+func (h *Handler) processImageCondition(cfg *v1.Config) error {
+	l, err := h.configmapclientwrapper.List()
+	if err != nil {
+		return err
+	}
+
+	list := []*corev1.ConfigMap{}
+	for _, cm := range l {
+		if cm.Name == util.IST2ImageMap {
+			continue
+		}
+		list = append(list, cm)
+	}
+
+	prefix := ""
+	if list == nil || len(list) == 0 {
+		if h.upsertInProgress {
+			return fmt.Errorf("configmap events because bulk imagestream upserts are still in progress")
```

Comment:
fyi ... this produces the following message in the pod log:

`unable to sync: configmap events because bulk imagestream upserts are still in progress, requeuing`